protobuf/protoc-gen-govalidator/plugin/plugin.go

```diff
@@ -746,6 +750,16 @@ func genMsgValidator(g *protogen.GeneratedFile, f *protogen.Field, varName strin
 	}
 }
 
+func genEnumValidator(g *protogen.GeneratedFile, f *protogen.Field, varName string) {
+	if !getBoolExtension(f, validator.E_EnumRequired) {
+		return
+	}
+
+	g.P("if int(", varName, ") == 0 {")
+	g.P("return ", fmtPackage.Ident("Errorf"), "(\"field ", f.Desc.Name(), " must be specified\")")
```

Comment:
If we do pass in the `UNSPECIFIED` enum value, this error message reads 
```
field enum must be specified
```
which can be considered as an incorrect error message.

Can we fix up this error message to explicitly say the value should be a non-zero value?